src/OnlineSales/Program.cs

```diff
@@ -272,4 +275,18 @@ private static void ConfigureEmailServices(WebApplicationBuilder builder)
 
         // builder.Services.AddScoped<IEmailFromTemplateService, EmailFromTemplateService>();
     }
+
+    private static void ConfigureCORS(WebApplicationBuilder builder)
+    {
+        builder.Services.AddCors(options =>
+        {
+            options.AddDefaultPolicy(policy =>
+            {
+                policy
+                    .AllowAnyOrigin()
```

Comment:
It is a very bad idea to allow any origin - very big security vulnerability. Please make a configurable list of trusted origins.